pkg/apis/migration/v1alpha1/owner_reference.go

```diff
@@ -0,0 +1,34 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+func SetOwnerReference(owner metav1.Object, ownerType runtime.Object, dim metav1.Object) {
```

Comment:
@alaypatel07 could we name `dim metav1.Object` more generically since this looks like it will work on any metav1.Object?

Reply:
since we already have an owner, do you mind if I generically name this `obj`?

Reply:
Sure, just something other than DIM